packages/cxl-ui/src/components/cxl-course-card.js

```diff
@@ -25,6 +25,22 @@ export class CXLCourseCardElement extends CXLBaseCardElement {
     this[`_${name}HasChildren`] = !!children.length;
   }
 
+  _renderCta() {
+    return html`
+      <vaadin-button
+        class="cta"
+        theme="tertiary"
+        @click=${(e) => {
```

Comment:
We should move away from the technique of using JS to update location. It doesn't allow us to track CTA performance.

https://cxlworld.slack.com/archives/D01JNHHHA1H/p1685977708409489